### PR TITLE
Simplified development environment

### PR DESCRIPTION
**Pull request by tomaszwozniak**
- Updated dependencies
- Replaced posgresql to sqlite
- One command for virtualenv, python packages, migration and initial gulp tasks.

### CODE REVIEW

File: bin/install_dependencies [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+BASEDIR=`dirname $0`/..
+
+apt-get update
+apt-get build-dep -y python-imaging
+apt-get install -y python-virtualenv libjpeg8 libjpeg62-dev libfreetype6 libfreetype6-dev npm
+cd $BASEDIR/apps/volontulo && npm install --global gulp-cli
```
Comment (magul) [inline bin/install_dependencies]: There is no need to go to `apps/volontulo` to execute `npm install gulp-cli`. The only reason to go there is if You're going to use `package.json`, which is placed there.

File: bin/install_dependencies [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+BASEDIR=`dirname $0`/..
```
Comment (magul) [inline bin/install_dependencies]: If we will not change directory below, this variable is not required here.

File: etc/local_config.yaml.sample [hunk 3]
```diff
@@ -14,3 +14,5 @@ db_port:
 db_name:
 db_user:
 db_pass:
+
```
Comment (magul) [inline etc/local_config.yaml.sample:17]: Do we need this lines?

File: bin/install_dependencies [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+apt-get update
+apt-get build-dep -y python-imaging
+apt-get install -y python-virtualenv libjpeg8 libjpeg62-dev libfreetype6 libfreetype6-dev npm
```
Comment (magul) [inline bin/install_dependencies:5]: Maybe we should point directrly `node` here. I know, that `npm` required `node`, or more accurate will be to say, that `npm` is bundled with `node`, but for some reason (and I understand why) You're pointing, that `node` will be istalled in `README`.

**Comment by magul** [comment]
Beside few minor issues (or sugestions), great work! :+1:

**Comment by Yog** [comment]
So, we switch back to sqlite to make todays development simpler? ;D
Or more like, use sqlite in development and postgreSQL on production?

Eighter way i gues i'm going to test it todays 'morning'.

File: etc/local_config.yaml.sample (hunk 3 above)
Comment (tomaszwozniak) [inline etc/local_config.yaml.sample:17]: It will seperate configs added by bash >>

**Comment by tomaszwozniak** [comment]
@Yog I've try to simplify setup for hackaton purpose to use one service less. For know we don't use any special postgres functionality and regarding with issues we are not going to, so we can develop on SQlite. For production we will still use PostgreSQL.

**Comment by magul** [comment]
@tomaszwozniak as we spoke offline, we need to switch email backend to `FILEBACKEND` for development environments and also include admin fixture in setup. I will provide You this fixture in another comment.

**Comment by magul** [comment]
[fixtures.json](https://files.zenhub.io/56d02e855674421e486561c5)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)

fixture, that is creating admin user with his/her profile:
- user: admin@volontuloapp.org
- pass: stx123

**Comment by tomaszwozniak** [comment]
@magul I've updated initial account.

**Pull request merged**